mithril-common/src/entities/signed_entity_config.rs

```diff
@@ -89,11 +91,14 @@ impl SignedEntityConfig {
     pub fn list_allowed_signed_entity_types(
         &self,
         time_point: &TimePoint,
-    ) -> Vec<SignedEntityType> {
-        self.list_allowed_signed_entity_types_discriminants()
-            .into_iter()
-            .map(|discriminant| self.time_point_to_signed_entity(discriminant, time_point))
-            .collect()
```

Comment:
No need to change the code.
With the into_iter and the new signature, the Vec<Resutl<_>> is mapped to a Result<Vec<_>>

Reply:
Done ✅